draco_coffee.services.yml

```diff
@@ -0,0 +1,4 @@
+services:
+  draco_coffee.manager:
+    class: Drupal\draco_coffee\DracoCoffeeManager
```

Comment:
Is manager the right name? I usually assume managers are for plugin types (I know, except for EntityTypeManager). I guess the point is that it's better if classes represent something real, like "DracoCoffeePot"?

Reply:
Deal! Changing.

Reply:
Changed to DracoCoffeePot :coffee: